IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/NewsCollectionViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  NewsCollectionViewCell.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import SDWebImage
+
+final class NewsCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "NewsCollectionViewCell"
+    static let Height = 80.0
+    
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblDate: UILabel!
+    @IBOutlet weak var lblDescription: UILabel!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblAuthor: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.contentView.frame = CGRect(x: self.contentView.frame.origin.x, y: self.contentView.frame.origin.y, width:UIScreen.main.bounds.width , height: CGFloat(NewsCollectionViewCell.Height))
```

Comment:
Since after renaming the file, the comment is lost, I'll ask again. 

Do we need this line? The height should be resolved via `getHeightForRow`, and settings width to `UIScreen.main.bounds.width` shouldn't be required.